manifest.yml

```diff
@@ -1,15 +1,15 @@
 ---
 applications:
 - name: pulse
-  memory: 512M
+  memory: 768M
```

Comment:
This is the max we can fit for the production site. I'm not sure how we're hitting our 2GB quota in the org, given what I can see by inspecting the contents via `cf`, but this is the ceiling.